### PR TITLE
[Refactor](Export) delete useless code of Export

### PR DESCRIPTION
**Pull request by BePPPower**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java [hunk 1]
```diff
@@ -96,75 +90,16 @@ private void writeUnlock() {
     @Override
     public synchronized void start() {
         super.start();
-        exportingExecutor.start();
     }
 
     @Override
     protected void runAfterCatalogReady() {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java:96]: We don't need to extends from MasterDaemon any more

**Comment by BePPPower** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java (hunk 1 above)
Comment (BePPPower) [reply fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java]: done

**Review by morningman — APPROVED** [review]
LGTM

**Review by wsjz — APPROVED** [review]

**Review by wsjz — APPROVED** [review]
LGTM

**Comment by wsjz** [comment]
这是来自QQ邮箱的假期自动回复邮件。
好，我最近正在休假中

 
您，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Pull request merged**